src/Microsoft.VisualStudio.Editor.Razor/DefaultImportDocumentManager.cs

```diff
@@ -112,11 +113,18 @@ public override void OnUnsubscribed(VisualStudioDocumentTracker tracker)
             }
         }
 
-        private IEnumerable<RazorProjectItem> GetImportItems(VisualStudioDocumentTracker tracker)
+        private IEnumerable<RazorSourceDocument> GetImportDocuments(VisualStudioDocumentTracker tracker)
         {
             var projectDirectory = Path.GetDirectoryName(tracker.ProjectPath);
-            var templateEngine = _templateEngineFactoryService.Create(projectDirectory, _ => { });
-            var imports = templateEngine.GetImportItems(tracker.FilePath);
+            var projectEngine = _projectEngineFactoryService.Create(projectDirectory, _ => { });
+            var importFeature = projectEngine.Features.OfType<IRazorImportFeature>().FirstOrDefault();
+
+            if (importFeature == null)
+            {
+                Debug.Fail("The import feature should always be available on the project engine.");
+                return Enumerable.Empty<RazorSourceDocument>();
+            }
+            var imports = importFeature.GetImports(tracker.FilePath);
```

Comment:
Hmm, actually this strips imports that don't exist on disk. Will have to use the options + project to make this one work the same.